Sources/DistributedActors/Time.swift

```diff
@@ -398,7 +212,7 @@ extension Deadline {
         self.hasTimeLeft(until: .now())
     }
 
-    public func hasTimeLeft(until: Deadline) -> Bool {
+    public func hasTimeLeft(until: ContinuousClock.Instant) -> Bool {
         !self.isBefore(until)
     }
```

Comment:
I like those... let's keep -- I wonder if people might be upset if we make them public though 🤔

Reply:
Made these internal